blueprints/blueprint-test/files/__root__/__name__-test.js

```diff
@@ -1,21 +1,24 @@
 'use strict';
 
-const blueprintHelpers = require('ember-cli-blueprint-test-helpers/helpers');
-const setupTestHooks = blueprintHelpers.setupTestHooks;
-const emberNew = blueprintHelpers.emberNew;
-const emberGenerateDestroy = blueprintHelpers.emberGenerateDestroy;
+const { 
+  setupTestHooks,
+  emberNew,
+  emberGenerateDestroy,
+} = require('ember-cli-blueprint-test-helpers/helpers');
 
-const expect = require('ember-cli-blueprint-test-helpers/chai').expect;
+const {
+  expect
+} = require('ember-cli-blueprint-test-helpers/chai');
 
 describe('Acceptance: ember generate and destroy <%= blueprintName %>', function() {
   setupTestHooks(this);
 
   it('<%= blueprintName %> foo', function() {
-    let args = ['<%= blueprintName %>', 'foo'];
+    const args = ['<%= blueprintName %>', 'foo'];
```

Comment:
Can you change this to `let` please?

Reply:
👍

Reply:
But just a small question regarding this.  Why not use "const"?  Isn't it encouraged to use "const"?

Reply:
This is a topic of debate, but the Ember community tends to lean towards `let`. See https://jamie.build/const and https://madhatted.com/2016/1/25/let-it-be for more info.